### PR TITLE
fixed single line if statemetns by adding curly braces, removed comme…

### LINKED ISSUE #530: Remove Commented out Code and fix if format
**Summary**
- [x] swirlds/hedera-fpcomplete-audit#142

- [x]  https://github.com/swirlds/hedera-fpcomplete-audit/pull/161

- [x] https://github.com/swirlds/hedera-fpcomplete-audit/pull/173

- [x] https://github.com/swirlds/hedera-fpcomplete-audit/pull/169

**Possible resolution**
Remove all commented out code from the project.
Fix single line if statement formatting
Use BigInteger to calculate total account balance and validate it on startUp
Split the echo command to separate string and array.

### PR DESCRIPTION
**Pull request by anighanta**
…nted out code in TransactionSigner

Signed-off-by: Anirudh Ghanta <anirudh.ghanta@hedera.com>

**Related issue(s)**:
Closes #530 

**Summary of the change**:
Removed commented out code. 
Fixed if format by using intelliJ's code formatting rule.
Use BigInteger to calculate total account balance and validate it on startUp.
Split the echo command to separate string and array.

**External impacts**:
None.

**Applicable documentation**
- [ ] Javadoc
- [ ] HAPI protobuf comments
- [ ] README

### CODE REVIEW

File: hapi-proto/src/main/java/com/hedera/services/legacy/proto/utils/KeyExpansion.java [hunk 1]
```diff
@@ -640,7 +640,9 @@ public static void expandKeyMinimum4Signing(Key key, int depth, List<Key> expand
         int i = 0;
         for (Key aKey : tKeys) {
           if(i++ >= thd) // if threshold is reached, stop expanding keys
-            break;
+		  {
+			  break;
```
Comment (ljianghedera) [inline hapi-proto/src/main/java/com/hedera/services/legacy/proto/utils/KeyExpansion.java:644]: Should we throw some exception to alert this situation. Or at least log a warning message?

File: hapi-proto/src/main/java/com/hederahashgraph/fee/FeeBuilder.java [hunk 2]
```diff
@@ -449,13 +424,15 @@ public static int getBaseTransactionRecordSize(TransactionBody txBody) {
...
           + txBody.getCryptoTransfer().getTransfers().getAccountAmountsCount()
-          * (BASIC_ACCT_AMT_SIZE);
+          * (BASIC_ACCOUNT_AMT_SIZE);
     }
     return txRecordSize;
   }
 
   public static long getTxRecordUsageRBH(TransactionRecord txRecord, int timeInSeconds) {
-	if(txRecord == null) return 0;
+	if(txRecord == null) {
```
Comment (Neeharika-Sompalli) [inline hapi-proto/src/main/java/com/hederahashgraph/fee/FeeBuilder.java:433]: Need to check code alignment in all the modified files. In many files the code is not aligned.

File: hapi-proto/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java [hunk 3]
```diff
@@ -343,20 +340,12 @@ private int getCryptoTransferBodyTxSize(TransactionBody txBody) {
...
-   * txBody.getCryptoTransfer().getTransfers(); List<AccountAmount> accountAmounts =
-   * transferList.getAccountAmountsList(); for (AccountAmount actAmt : accountAmounts) { if
-   * (actAmt.getAmount() > 0) { amount = amount + actAmt.getAmount(); } } return Math.round(amount /
-   * 1000); }
-   */
-
-  ////////////////////////////////////////////////////////////////////////// Query Fee
-  ////////////////////////////////////////////////////////////////////////// //////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////// Query Fee //////////////////////////////////////////////////
+  //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (ljianghedera) [inline hapi-proto/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java:348]: These are rather inconsistent. Probally change to either a simple single line comment // or  multi-line /* ... */ comment.

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java [hunk 4]
```diff
@@ -245,13 +258,16 @@ public void updateAccountWithMasterAsPayer() throws Throwable {
           log.info("update failed: payer=" + payerID.getAccountNum() + ", target acount=" + account);
         }
         else if(account.getAccountNum() >= 51 && account.getAccountNum() <= 80)  // master can update protected accounts 51 through 80 without needing sig of updated account
-          updateAccount(account, payerID, nodeID, ResponseCodeEnum.OK, ResponseCodeEnum.SUCCESS, false, null);
-        else if(account.getAccountNum() <= 1000) { // master can NOT update protected accounts other than 51 through 80
+		{
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:261]: This brace should be next to `else if {`

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java [hunk 5]
```diff
@@ -245,13 +258,16 @@ public void updateAccountWithMasterAsPayer() throws Throwable {
...
+			updateAccount(account, payerID, nodeID, ResponseCodeEnum.OK, ResponseCodeEnum.SUCCESS, false, null);
+		} else if(account.getAccountNum() <= 1000) { // master can NOT update protected accounts other than 51 through 80
           updateAccount(account, payerID, nodeID, ResponseCodeEnum.AUTHORIZATION_FAILED, null, false, null);
           log.info("update failed: payer=" + payerID.getAccountNum() + ", target acount=" + account);
         }
         else // can update non-protected accounts but need sig of the existing account
-          updateAccount(account, payerID, nodeID, ResponseCodeEnum.OK, ResponseCodeEnum.SUCCESS, true, null);
+		{
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:268]: same here

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java [hunk 6]
```diff
@@ -292,7 +309,9 @@ else if(account.getAccountNum() <= 1000) { // can NOT update protected accounts
...
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:312]: same here

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java [hunk 7]
```diff
@@ -333,7 +353,9 @@ else if(account.getAccountNum() <= 1000) { // can NOT update protected accounts
...
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:356]: same here

File: test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java [hunk 8]
```diff
@@ -76,13 +76,17 @@ public static void runMultiThreadedTests(String host, int port, long nodeAccount
         int markerWindow, long retryFreq, boolean isExponentialBackoff) throws Throwable {
 
     if(op_batchStr != null) // use command line to config test
-      initTest(op_batchStr, threadCounts, maxTpsDesiredPerThread);
-    else // use umbrellaTest.properties to config
-      initTest();
+	{
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java:79]: same here

File: test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java [hunk 9]
```diff
@@ -76,13 +76,17 @@ public static void runMultiThreadedTests(String host, int port, long nodeAccount
...
+		initTest(op_batchStr, threadCounts, maxTpsDesiredPerThread);
+	} else // use umbrellaTest.properties to config
+	{
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java:82]: same here

File: hapi-proto/src/main/java/com/hederahashgraph/fee/FeeBuilder.java (hunk 2 above)
Comment (anighanta) [reply hapi-proto/src/main/java/com/hederahashgraph/fee/FeeBuilder.java:433]: @Neeharika-Sompalli will created a separate issue to track this for another PR https://github.com/hashgraph/hedera-services/issues/582

Comment (Neeharika-Sompalli) [reply hapi-proto/src/main/java/com/hederahashgraph/fee/FeeBuilder.java:433]: Sounds good. Thanks @anighanta !

File: hapi-proto/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java (hunk 3 above)
Comment (anighanta) [reply hapi-proto/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java:348]: Yeah.. changing it to a single line comment. I think it is just put in place to denote the start of query fee calculation methods.

File: hedera-node/src/test/java/com/hedera/services/legacy/unit/handler/FileServiceHandler.java [hunk 10]
```diff
@@ -146,11 +146,12 @@ public static boolean hasAuthorityToUpdate(AccountID account, Object entity) {
           isMasterAccount((AccountID) entity)) { // master account cannot update itself
         return false;
       }
-      else if (seq >= 50 && seq <= 80)
+      else if (seq >= 50 && seq <= 80) {
```
[the remark this answers is not in the document]
Comment (anighanta) [reply hedera-node/src/test/java/com/hedera/services/legacy/unit/handler/FileServiceHandler.java:149]: makes sense, thank you.

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java (hunk 4 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:261]: Updated, Thanks

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java (hunk 5 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:268]: updated, thanks.

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java (hunk 6 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:312]: updated, thanks.

File: test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java (hunk 7 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java:356]: updated, thanks.

File: test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java (hunk 8 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java:79]: updated, thanks.

File: test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java (hunk 9 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/regression/umbrella/ThreadPerAPITest.java:82]: updated, thanks.

File: hapi-proto/src/main/java/com/hedera/services/legacy/proto/utils/KeyExpansion.java (hunk 1 above)
Comment (anighanta) [reply hapi-proto/src/main/java/com/hedera/services/legacy/proto/utils/KeyExpansion.java:644]: Makes sense. Adding a debug level log here.

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM

File: hedera-node/src/main/java/com/hedera/services/legacy/export/AccountBalanceExport.java [hunk 11]
```diff
@@ -142,12 +143,12 @@ public String exportAccountsBalanceCSVFormat(
     if(log.isDebugEnabled()){
       log.debug("Size of accountMap :: {}", accountMap.size());
     }
-    long totalBalance = 0L;
+    BigInteger totalBalance = BigInteger.ZERO;
```
Comment (ljianghedera) [inline hedera-node/src/main/java/com/hedera/services/legacy/export/AccountBalanceExport.java:146]: Why do we want to change the totalBalance's type from long to BigInteger? I thought total balance is 5_000_000_000_000_000_000, the range of long is large enough to cover that. Changing from long to BigInteger will have some performance impact.

File: test-clients/src/main/java/com/hedera/services/legacy/netty/MultiAccountTransferSeqThread.java [hunk 12]
```diff
@@ -337,25 +330,22 @@ public void doOneIteration(int _acItr)
...
           {
           log.error("Got Bad Precheck Stuff ** Adding to Retry Queue ** " + transferRes
               .getNodeTransactionPrecheckCode());
           if(transferRes.getNodeTransactionPrecheckCode() == ResponseCodeEnum.TRANSACTION_OVERSIZE){
             badTransList.add(transfer1);
           }
-          else totalBadReceipts++;
-         // totalBadReceipts++;
+          else {
+          }
```
Comment (ljianghedera) [inline test-clients/src/main/java/com/hedera/services/legacy/netty/MultiAccountTransferSeqThread.java:348]: Should the change be like this:

```
else {
    totalBadReceipts++;
}
```

File: hedera-node/src/main/java/com/hedera/services/legacy/export/AccountBalanceExport.java (hunk 11 above)
Comment (anighanta) [reply hedera-node/src/main/java/com/hedera/services/legacy/export/AccountBalanceExport.java:146]: https://github.com/swirlds/hedera-fpcomplete-audit/pull/173

File: test-clients/src/main/java/com/hedera/services/legacy/smartcontract/V3V4Before.java [hunk 13]
```diff
@@ -135,27 +135,10 @@ private void loadGenesisAndNodeAcccounts() throws Exception {
...
-		 /* Transaction transferTx = RequestBuilder.getCryptoTransferRequest(
-					 payer.getAccountNum(), payer.getRealmNum(), payer.getShardNum(), nodeAccount.getAccountNum(), nodeAccount.getRealmNum(), nodeAccount.getShardNum(),
-					 50, timestamp, transactionDuration, false, "test", sigList,
-					 payer.getAccountNum(), -100l, nodeAccount.getAccountNum(), 100l);*/
-
-    Transaction transferTx = TestHelper.createTransfer(payer, accountKeys.get(payer).get(0),
+  private Transaction createQueryHeaderTransfer(AccountID payer, long transferAmt) {
+    return TestHelper.createTransfer(payer, accountKeys.get(payer).get(0),
         nodeAccount, payer,
         accountKeys.get(payer).get(0), nodeAccount, transferAmt);
```
Comment (ljianghedera) [inline test-clients/src/main/java/com/hedera/services/legacy/smartcontract/V3V4Before.java:141]: Are we creating same transfer transaction as original code? For example, the duration of transaction in new code is missing.

File: test-clients/src/main/java/com/hedera/services/legacy/netty/MultiAccountTransferSeqThread.java (hunk 12 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/netty/MultiAccountTransferSeqThread.java:348]: This counter is no more. Removing this empty else block

File: test-clients/src/main/java/com/hedera/services/legacy/smartcontract/V3V4Before.java (hunk 13 above)
Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/legacy/smartcontract/V3V4Before.java:141]: I just removed the unused code. I you look at the old code you can see that we are not using the transaction duration and the siglist.

Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/legacy/smartcontract/V3V4Before.java:141]: Got it.

**Review by ljianghedera — APPROVED** [review]
LGTM.

**Review by qnswirlds — COMMENTED** [review]
LGTM in general. I think Leo did a more thorough review, which should be sufficient. Please finalize this PR and merge it to master asap. We need to merge master to another branch the earlier the better for the release of v0.9.0. Thanks!

**Pull request merged**